cosmetics-web/app/forms/notification_wizard/delete_component_form.rb

```diff
@@ -0,0 +1,19 @@
+module NotificationWizard
+  class DeleteComponentForm < Form
+    attribute :component_id
+    attribute :notification
+
+    validates :component_id, presence: true
+
+    def delete
+      return false unless valid?
+      raise("Can not remove item") if notification.components.count < 3
+      raise ActiveRecord::RecordNotFound if %w[notification_complete deleted].include?(notification.state)
```

Comment:
Suggestion: replace hardcoded states for constants or the state helper methods:
```suggestion
      raise ActiveRecord::RecordNotFound if notification.notification_completed? || notification.deleted?
```